pkg/standalone/run.go

```diff
@@ -25,127 +26,250 @@ const sentryDefaultAddress = "localhost:50001"
 
 // RunConfig represents the application configuration parameters.
 type RunConfig struct {
-	AppID              string
-	AppPort            int
-	HTTPPort           int
-	GRPCPort           int
-	ConfigFile         string
-	Protocol           string
+	AppID              string `env:"APP_ID" arg:"app-id"`
+	AppPort            int    `env:"APP_PORT" arg:"app-port"`
+	HTTPPort           int    `env:"DAPR_HTTP_PORT" arg:"dapr-http-port"`
+	GRPCPort           int    `env:"DAPR_GRPC_PORT" arg:"dapr-grpc-port"`
+	ConfigFile         string `arg:"config"`
+	Protocol           string `arg:"app-protocol"`
 	Arguments          []string
-	EnableProfiling    bool
-	ProfilePort        int
-	LogLevel           string
-	MaxConcurrency     int
-	PlacementHostAddr  string
-	ComponentsPath     string
-	AppSSL             bool
-	MetricsPort        int
-	MaxRequestBodySize int
+	EnableProfiling    bool   `arg:"enable-profiling"`
+	ProfilePort        int    `env:"DAPR_PROFILE_PORT" arg:"profile-port"`
+	LogLevel           string `arg:"log-level"`
+	MaxConcurrency     int    `arg:"app-max-concurrency"`
+	PlacementHostAddr  string `env:"HOST_ADDRESS" arg:"placement-host-address"`
```

Comment:
@anhldbk See lines https://github.com/dapr/cli/blob/master/cmd/run.go#L62 and https://github.com/dapr/cli/blob/master/cmd/run.go#L81.

When the Env var is given as DAPR_PLACEMENT_HOST_ADDRESS, it is picked up as placement-host-address.


@yaron2 Do you think this change is needed?

Reply:
@mukundansundar actually there's no such env DAPR_PLACEMENT_HOST_ADDRESS right? I'm gonna remove it

Reply:
The prefix is `DAPR` for the viper bindings ... the PLACEMENT_HOST_ADDRESS is something that is being used right now so should not be removed ... It is translated to `placement-host-address` by `viper`

Reply:
@mukundansundar Correct me if I'm wrong: all available env variables are defined here: https://github.com/dapr/dapr/blob/master/pkg/config/env/env_variables.go

Reply:
@anhldbk There are certain env vars for Dapr CLI alone that will not be defined in that file ... Some are available here https://docs.dapr.io/reference/environment/ ... Some more arguments and annotations supported by daprd, dapr CLI or k8s are here https://docs.dapr.io/reference/arguments-annotations-overview/

Reply:
@mukundansundar Thanks for your pointer. Now I see we have the inconsistency between our docs & code. I'll add a todo in my checklist to fix it soon.

To clarify it a little bit for field `PlacementHostAddr`, we'll have:

- env variable: `DAPR_PLACEMENT_HOST`
- argument: `--placement-address` (not as the currently used `placement-host-address`). See [your link](https://docs.dapr.io/reference/arguments-annotations-overview/)

Correct?

Reply:
@anhldbk Sorry for the late reply, if you see `dapr run --help` as well as the `daprd` arguments, the field name has been changed to `--placement-host-address` the change needs to be done in docs. 
The Environment variable as a consequence will be `DAPR_PLACEMENT_HOST_ADDRESS`